scrowl-io/owl-ui
# [sc-292] - Create Button Component

### Short summary
A default `Button` component was created to render buttons and links.

Three variants were created:
- Default
- Primary
- Link

*The designs were not provided yet for each variant so basic styles were added to get the component ready to receive the styles that will be provided.
**The button with icons will be added once we get the Icon component created. 

---

### Test steps
1. Run `yarn start:storybook`
2. Navigate through the different variants and ensure no errors pop up in the console.
